CMakeLists.txt

```diff
@@ -253,14 +274,20 @@ if (NOT EMSCRIPTEN)
     link_libraries(gcov)
   endif ()
 
+  set(OMR_GC OFF)
+  set(OMR_OMRSIG OFF)
+  set(OMR_JIT ON)
+  set(OMR_JITBUILDER ON)
+  add_subdirectory(third_party/omr)
```

Comment:
I'm getting spammed with tons of warnings when compiling the OMR code due to the extra warning flags set by WABT. We should probably consider at least disabling some of the more spammy warning flags (e.g. `-Wreorder` and `-Wold-style-cast`) when compiling OMR to avoid ridiculous amounts of spam when building.

Reply:
OMR project hygiene request: would you please capture the list and post in an issue at Eclipse OMR?

I can't promise anyone will work on it quickly, but I would like to improve this situation.